src/client/pages/AssessmentSection/DataTable/DataTable.tsx

```diff
@@ -52,6 +53,13 @@ const DataTable: React.FC<Props> = (props) => {
   //   (state) => odp && !disabled && Objects.isFunction(canGenerateValues) && canGenerateValues(state)
   // )
 
+  useListenNodeUpdates({
```

Comment:
maybe better to add the listener in AssessmentSection? the nodeUpdates are not table specific .